packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -3920,8 +3920,25 @@ export class ITxnSubmit extends Op {
       );
     }
 
+    // initial contract account.
+    const appID = this.interpreter.runtime.ctx.tx.apid;
+    const contractAddress = getApplicationAddress(appID as number);
+    const contractAccount = {
+      addr: contractAddress,
+      sk: Buffer.from([])
+    };
```

Comment:
can't we use `getAccount` here? I am assuming application is already deployed at this point, so the contract account must be present in runtime's context. right?

Reply:
Yes, I will use runtime.getAccount in this case.